### PR TITLE
feature/update image list example

### PR DESCRIPTION
**Pull request by a-tarasyuk**

### CODE REVIEW

File: samples/ImageList/src/controls/SearchField.tsx [hunk 1]
```diff
@@ -22,14 +22,14 @@ const _styles = {
...
         padding: 5,
         margin: 10,
-        height: 30
+        height: 30,
     }),
 };
 
 export class SearchField extends RX.Component<SearchFieldProps, SearchFieldState> {
-    readonly state: SearchFieldState = { value: '' };
+    public readonly state: SearchFieldState = { value: '' };
```
Comment (erictraut) [inline samples/ImageList/src/controls/SearchField.tsx]: In TypeScript, "public" is the default modifier. For the coding style guideline we use in ReactXP, we don't include "public". The access modifier is included only if it's "protected" or "private". Please remove these public modifiers.

File: samples/ImageList/src/stores/ImageStore.ts [hunk 2]
```diff
@@ -23,26 +23,26 @@ export class ImageStore extends StoreBase {
     private _request: SyncTasks.STPromise<void> | null = null;
 
     @autoSubscribe
-    getImages() {
+    public getImages() {
```
Comment (erictraut) [inline samples/ImageList/src/stores/ImageStore.ts]: See comment above about "public".

File: samples/ImageList/src/views/ImageList.tsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ const _styles = {
         backgroundColor: '#fff',
         flexDirection: 'column',
         alignSelf: 'stretch',
-        margin: 0
+        margin: 0,
```
Comment (erictraut) [inline samples/ImageList/src/views/ImageList.tsx]: In the coding style that we use consistently throughout the ReactXP source base, we don't allow dangling commas. Please remove.

File: samples/ImageList/src/views/RootView.tsx [hunk 4]
```diff
@@ -22,12 +22,12 @@ const _styles = {
         flex: 1,
     }),
     statusSpacer: RX.Styles.createViewStyle({
-        marginTop: 22
-    })
+        marginTop: 22,
+    }),
```
Comment (erictraut) [inline samples/ImageList/src/views/RootView.tsx]: See above about dangling comma and public modifier.

**Comment by a-tarasyuk** [comment]
@erictraut thanks!

**Pull request merged**